src/Lykke.Job.TxDetector/Commands/ProcessTransactionCommand.cs

```diff
@@ -7,5 +8,7 @@ public class ProcessTransactionCommand
     {
         [ProtoMember(1)]
         public string TransactionHash { get; set; }
+        [ProtoMember(2)]
+        public IBlockchainTransaction BlockchainTransaction { get; set; }
```

Comment:
There is no need to sending full transaction DTO. The consumer can retrieve all needed information by transaction hash.